pipelines/ApVerifyWithFakes.yaml

```diff
@@ -0,0 +1,49 @@
+# Gen 3 pipeline for ap_verify
+# This concatenates various lsst.verify metrics to an AP pipeline
+
+description: Fully instrumented AP pipeline
+imports:
+    - location: $AP_PIPE_DIR/pipelines/ApPipe.yaml
+    - location: $AP_VERIFY_DIR/pipelines/MetricsRuntime.yaml
+    - location: $AP_VERIFY_DIR/pipelines/MetricsMiscWithFakes.yaml
+tasks:
+    createFakes:
+        class: lsst.ap.pipe.createApFakes.CreateRandomApFakesTask
+        config:
+            magMin: 20
+            magMax: 26
+            fraction: 0
+            fakeDensity: 2000
+    coaddFakes:
+        class: lsst.pipe.tasks.insertFakes.InsertFakesTask
+        config:
+            doSubSelectSources: True
+            sourceSelectionColName: "isTemplateSource"
+    visitFakes:
+        class: lsst.pipe.tasks.processCcdWithFakes.ProcessCcdWithFakesTask
+        config:
+            insertFakes.doSubSelectSources: True
+            insertFakes.sourceSelectionColName: "isVisitSource"
+    imageDifference:
+        class: lsst.pipe.tasks.imageDifference.ImageDifferenceTask
+        config:
+            # Always prefer decorrelation; may eventually become ImageDifferenceTask default
+            doDecorrelation: True
+            detection.thresholdValue: 5.0  # needed with doDecorrelation
+            # Don't have source catalogs for templates
+            doSelectSources: False
+            connections.fakesType: "fakes_"
+    diaPipe:
+        # TODO: how to prevent duplication with ApPipe definition?
+        class: lsst.ap.association.DiaPipelineTask
+        config:
+            apdb.isolation_level: READ_UNCOMMITTED
+            doPackageAlerts: True
+            doWriteAssociatedSources: True
```

Comment:
I wasn't aware of this flag. Is it something we should be using in the main `ap_verify` pipeline?

Reply:
Not really. It's basically to write out the intermediate data product that ends up being written into the Apdb. Right now it's used pretty exclusively for the fakes task post processing steps. Might have another use later but not for now.